xml/System.IO.IsolatedStorage/IsolatedStorageFileStream.xml

```diff
@@ -1913,8 +1926,8 @@ Dim source As New IsolatedStorageFileStream(UserName,FileMode.Open,isoFile)
         <Parameter Name="buffer" Type="System.ReadOnlySpan&lt;System.Byte&gt;" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="buffer">The read-only byte span from which to copy bytes to the current isolated storage file stream.</param>
+        <summary>Writes a block of bytes to the isolated storage file stream object using data read from the buffer.</summary>
```

Comment:
It would be useful to disambiguate the two Write overloads (the second on line 1977). Perhaps "a buffer consisting of a read-only byte span" and "a buffer consisting of a byte array"?

Reply:
```suggestion
        <summary>Writes a block of bytes to the isolated storage file stream object using data read from a buffer consisting of a read-only byte span.</summary>
```